e2e/cypress/integration/files_and_attachments/upload_files_spec.js

```diff
@@ -179,6 +179,28 @@ describe('Upload Files', () => {
             cy.wait(TIMEOUTS.HALF_SEC);
         });
     });
+
+    it('MM-T337 CTRL/CMD+U - Five files on one message, thumbnails while uploading', () => {
+        cy.visit(`/${testTeam.name}/channels/town-square`);
+        const filename = 'huge-image.jpg';
+        cy.get('#centerChannelFooter').find('#fileUploadInput').
+            attachFile(filename).
+            attachFile(filename).
+            attachFile(filename).
+            attachFile(filename).
+            attachFile(filename);
+        for (let i = 1; i < 5; i++) {
```

Comment:
Should this be `i < 4`? I'm getting error on 4th child since it's not visible.
<img width="528" alt="Screen Shot 2020-09-02 at 4 04 02 PM" src="https://user-images.githubusercontent.com/5334504/91955601-079d7d00-ed36-11ea-9be1-4d10d0bd1888.png">

<img width="837" alt="Screen Shot 2020-09-02 at 4 03 24 PM" src="https://user-images.githubusercontent.com/5334504/91955568-f94f6100-ed35-11ea-95f2-9b9eb533f427.png">